engine-nextgen/src/main/kotlin/graphql/nadel/NextgenEngine.kt

```diff
@@ -273,6 +301,24 @@ class NextgenEngine @JvmOverloads constructor(
         )
     }
 
+    private fun usesDynamicServiceResolution(
```

Comment:
Particularly this code doesn't really belong here. It should be in a sort of `DynamicServiceResulotion` object/class.